AutomatticTracks/src/main/java/com/automattic/android/tracks/MessageBuilder.java

```diff
@@ -105,8 +105,8 @@ public static synchronized JSONObject createEventJSONObject(Event event, JSONObj
                 eventJSON.put(USER_TYPE_KEY, USER_TYPE_ANON);
                 eventJSON.put(USER_ID_KEY, event.getUser());
             } else {
+                eventJSON.put(USER_TYPE_KEY, event.getUserType().name().toLowerCase());
```

Comment:
> wpcom:user:id, instead we're sending wpcom

Ah, see I thought I probably broke it ;) Thanks.

+1 for a switch statement! I'll change it to that